src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -246,6 +246,18 @@ To set attributes for the HTML `body` tag use the `<attribute>` instruction.
 </page>
 ```
 
+Also from admin site __Layout Update XML__ field of __Design__ section on _Content Edit_ page  set attributes for the HTML `body` tag use the `<attribute>` instruction.
+
+**Example:** Add a new class to the `body` tag.
+
+```xml
+    <body>
+        <attribute name="class" value="my-new-body-class"/>
+    </body>
+```
+
+![Block Class]({{ site.baseurl }}/common/images/body-class-result.png)
```

Comment:
Move the image after the original example for adding a new class to the body tag-- after line 237.